README.md

```diff
@@ -6,30 +6,30 @@ This repository contains public maps and active rotation list that are found on
-  * If the world version is from a future (1.9.x and above) or from past (1.7.x and below) versions of Minecraft, you **must** convert the map to be compatible with 1.8.9.
-    * This can be achieved manually or by using [a script to automate](https://github.com/mitchts/nbt-converter) the process.
+- Every map must work with Minecraft 1.8.9. To determine the version of the map, you can load it into your Singleplayer Menu.  
+- If the world version is from a future (1.9.x and above) or from past (1.7.x and below) versions of Minecraft, you **must** convert the map to be compatible with 1.8.9.  
+  - This can be achieved manually or by using [a script to automate](https://github.com/mitchts/nbt-converter) the process.  
 
 ### Chunk Pruning
 In order to remove undesired portions of the map as well as to decrease its total file size, the map needs to be pruned in order for it to be accepted into the repository.
-  * This means that **all** chunks excluding the map itself should be deleted from the world.
-  * This can be achieved with third-party programs such as [MCASelector](https://github.com/Querz/mcaselector), [MCEdit-Unified](https://github.com/Podshot/MCEdit-Unified), or [MCEdit 2.0](https://github.com/mcedit/mcedit2).
-  * **Note:** When using MCASelector, please make sure you select the `region` folder inside a world folder to edit a world, not the entire world folder.
-  * After pruning the map, please make sure that you only include the required world files (`region/*`, `level.dat`, `map.png`, `map.xml`, ). 
-    * For more information about cleaning the world files, [view this page on PGM Documentation](https://pgm.dev/docs/guides/packaging/cleaning-files).
+- This means that **all** chunks excluding the map itself should be deleted from the world.  
+- This can be achieved with third-party programs such as [MCASelector](https://github.com/Querz/mcaselector), [MCEdit-Unified](https://github.com/Podshot/MCEdit-Unified), or [MCEdit 2.0](https://github.com/mcedit/mcedit2).  
+- **Note:** When using MCASelector, please make sure you select the `region` folder inside a world folder to edit a world, not the entire world folder.  
+- After pruning the map, please make sure that you only include the required world files (`region/*`, `level.dat`, `map.png`, `map.xml`, ).  
+  - For more information about cleaning the world files, [view this page on PGM Documentation](https://pgm.dev/docs/guides/packaging/cleaning-files).  
 
 ### XML Creation
-  * All maps must include a valid `map.xml` to function properly with PGM. The map protocol must be `1.4.0` or above.
-  * The map version **must** follow the [Semantic Versioning](https://semver.org/) standard.
-  * Refer to [Additional Resources](#additional-resources) on following proper XML format.
+- All maps must include a valid `map.xml` to function properly with PGM. The map protocol must be `1.4.0` or above.  
```

Comment:
I thought going above proto version 1.4.0 was a 1.9 PGM thing?

Reply:
No, a good chunk of features supported by higher protocols have already been introduced to current PGM and work without trouble. Some OCC’s community-submitted maps (and some maps located at their new PublicMaps repo) are already using these protocols.